gatsby/content/blog/2022/06/2022-06-17-synapse-1.61.mdx

```diff
@@ -0,0 +1,72 @@
+---
+date: '2022-06-17'
+title: Synapse 1.61 released
+categories:
+- Releases
+author: Brendan Abolivier
+---
+
+Hey everyone! Guess what? [Synapse
+1.61](https://github.com/matrix-org/synapse/releases/tag/v1.61.0) is out! Let's
+have a look at it.
+
+## Farewell, groups
+
+If you are new to Matrix, you might have not heard of the feature referred to as
+"groups" or "communities" (depending on the context). This feature allowed
+grouping rooms and users to better represent a community, one of which being
+`+matrix:matrix.org` which used to represent the Matrix community. This may
+sound similar to [Matrix
+spaces](https://matrix.org/blog/2021/05/17/the-matrix-space-beta), and it would
+make sense since spaces is meant to be a more powerful replacement to groups.
```

Comment:
```suggestion
grouping rooms and users to better represent a community, one of which being
`+matrix:matrix.org` which used to represent the Matrix community. This may
sound similar to [Matrix
Spaces](https://matrix.org/blog/2021/05/17/the-matrix-space-beta), and it would
make sense since Spaces are meant to be a more powerful replacement for groups.
```